infrastructure_files/setup.env.example

```diff
@@ -4,13 +4,16 @@
 
 # Dashboard domain. e.g. app.mydomain.com
 NETBIRD_DOMAIN=""
-# e.g. dev-24vkclam.us.auth0.com
-NETBIRD_AUTH0_DOMAIN=""
```

Comment:
we should maintain compatibility with users that want to update the script but didn't read the docs. In these cases, they will have an old config file with NETBIRD_AUTH0... variables